### PR TITLE
docs: update additional resource doc

### PR DESCRIPTION
**Pull request by Lou1415926**
This PR updates additional resource doc to 
1. include Aurora 
2. add more information about output

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: site/content/docs/developing/additional-aws-resources.en.md [hunk 1]
```diff
@@ -41,7 +41,9 @@ However, by default, Copilot will pass the `App`, `Env`, and `Name` [Parameters]
 If you need to access your [Resources](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/resources-section-structure.html) from your ECS task, make sure to:
 
 1. Define an [IAM ManagedPolicy](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-iam-managedpolicy.html) resource in your template that holds the permissions for your task and add an [Output](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/outputs-section-structure.html) so that the permission is injected to your ECS Task Role.
-2. Create an [Output](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/outputs-section-structure.html) for any value that you want to be injected as an environment variable to your ECS tasks.
+2. Define a [Security Group](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-security-group.html) in your template and add an [Output](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/outputs-section-structure.html). The security group will be automatically attached to your ECS service.
```
Comment (efekarakus) [inline site/content/docs/developing/additional-aws-resources.en.md]: What do you think of transforming these into bullet points:
* If you need to add additional policies to your ECS task role, you can define an [IAM Managed Policy] .... * If you need to add an additional security group to your ECS service, you can define ...
* If you'd like to inject a secret, you can define a ...

**Review by efekarakus — COMMENTED** [review]
Awesome, thank you so much for updating this page!
FYI @toricls

File: site/content/docs/developing/additional-aws-resources.en.md (hunk 1 above)
Comment (Lou1415926) [reply site/content/docs/developing/additional-aws-resources.en.md]: done!

**Review by efekarakus — APPROVED** [review]

**Review by SoManyHs — APPROVED** [review]
:shipit:

**Pull request merged**